translations/en-us.yml

```
@@ -1158,6 +1158,7 @@ registries:
         disciplines: Disciplines
         edit_field: 'Edit {field}'
         license: License
+        link_to_archived_copy: 'Link to archived copy'
```

Nitpick: Thinking link it might be better to say this so people know where their registration is archived before clicking the link. The only other place where IA is mentioned is in the submission process and so non-contributors won't necessarily have the context around `archived`.
```suggestion
        link_to_archived_copy: 'Link to Internet Archive copy'
```